src/components/scores/concept-coach-cell.cjsx

```diff
@@ -10,15 +10,49 @@ ConceptCoachCell = React.createClass
 
   mixins: [CellStatusMixin] # prop validation
 
+  getInitialState: ->
+    placement: 'right'
+
+  setPopoverPosition: ->
```

Comment:
Also, I was only ever able to get the popover to show up to the left of the icon, never to the right. When is this extra code needed (since we would always have student names to the left)?